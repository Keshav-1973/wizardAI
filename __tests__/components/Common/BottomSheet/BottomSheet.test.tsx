import React from 'react';
import renderer from 'react-test-renderer';
import BottomSheetHelper from '@components/Common/BottomSheet/BottomSheet';

// Mocking useKeyboardStatus hook
jest.mock('@components/Common/Hooks/useKeyBoardStatus', () => () => false);

// Mocking @gorhom/bottom-sheet module
jest.mock('@gorhom/bottom-sheet', () => ({
  __esModule: true,
  default: jest.fn(),
  BottomSheetView: jest.fn().mockImplementation(({children}) => children),
}));

describe('BottomSheetHelper', () => {
  it('matches snapshot', () => {
    const props = {
      children: <></>, // Provide any children here
      bottomSheetRef: jest.fn(), // Mock bottomSheetRef function
      getSheetStatus: jest.fn(), // Mock getSheetStatus function
    };

    const tree = renderer.create(<BottomSheetHelper {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
