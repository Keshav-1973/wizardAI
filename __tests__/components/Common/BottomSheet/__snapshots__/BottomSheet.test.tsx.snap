// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomSheetHelper matches snapshot 1`] = `null`;
